js/compiler/compiler_test.go

```diff
@@ -70,9 +70,11 @@ func TestCompile(t *testing.T) {
 		t.Parallel()
 		c := New(testutils.NewLogger(t))
 		src := `1+(function() { return 2; })()`
-		pgm, code, err := c.Compile(src, "script.js", true)
```

Comment:
Nit; As the method is name `Parse` now, should we rename the test name to `TestParse`? 🤔